src/Tribe/Assets.php

```diff
@@ -409,7 +384,24 @@ public function admin_enqueue_scripts() {
 				'localize' => [
 					[
 						'name' => 'Attendees',
-						'data' => function () use ( $config_data ) {
+						'data' => static function () {
+							$config_data = [
```

Comment:
avoid local var, just return `apply_filters( '', [....] );

Then this can also change to fn() instead of function